source/administration/changelog.md

```diff
@@ -259,6 +260,8 @@ Multiple setting options were added to ``config.json``. Below is a list of the a
 
 ## Release v5.31 - [ESR](https://docs.mattermost.com/administration/extended-support-release.html)
 
+- **v5.31.3, release day TBD**
+  - Investigating an issue where cluster handlers are not immediately registered after starting the server, which leads to issues such as job servers not scheduling jobs. [MM-34179](https://mattermost.atlassian.net/browse/MM-34179)
```

Comment:
Actually the jobs are not getting scheduled by the cluster lead. Because we don't initialize schedulers, cluster handlers etc until a `http` request hits to a node.
```suggestion
  - Investigating an issue where cluster handlers are not immediately registered after starting the server, which leads to issues such as jobs are not scheduled until a request hits to the cluster. [MM-34179](https://mattermost.atlassian.net/browse/MM-34179)
```